server/api/utils/oauth_helper.py

```diff
@@ -0,0 +1,85 @@
+import base64
+from google_auth_oauthlib.flow import Flow
+from django.conf import settings
+from Crypto.Cipher import AES
+
+from Crypto import Random
+IV = Random.new().read(AES.block_size)
+KEY = bytes(settings.CRYPTOGRAPHIC_KEY, 'utf-8')
+CIPHER = AES.new(KEY, AES.MODE_CFB, IV)
+
+GOOGLE_CREDENTIALS = {
+    "web": {
+        "client_id": settings.GOOGLE_CLIENT_ID,
+        "project_id": "hini-1530850449245",
+        "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+        "token_uri": "https://accounts.google.com/o/oauth2/token",
+        "auth_provider_x509_cert_url":
+            "https://www.googleapis.com/oauth2/v1/certs",
+        "client_secret": settings.GOOGLE_CLIENT_SECRET,
+        "redirect_uris": [
+            "http://proton-dev.com:5000/authorize",
+            "http://localhost:5000/authorize"
+        ],
+        "javascript_origins":
+            [
+                "http://localhost:5000",
+                "http://proton-dev.com:5000"
+            ]
+    }
+}
+
+
+FLOW = Flow.from_client_config(GOOGLE_CREDENTIALS,
+                               scopes=['https://www.googleapis.com/auth/calendar'],
+                               redirect_uri='http://localhost:8000/api/v1/oauthcallback')
```

Comment:
### Critical

This redirect_uri should be set in the settings.py because we won't be using localhost in prod and staging environment.